### DISCUSSION TITLE
migrate to Symfony 5.4

### DISCUSSION THREAD

**Discussion by mmoqiseh** [symfony/symfony#46660] [Q&A] [answered]
hi. i have been using symfony 3 for the last 2 years. now i want to start a project with symfony 5.4.
my first question is that... is fosuserbundle  works in symfony 5.4 or there is an alternative in new symfony 5.4 ?
and is this 5.4 version is recommended to use? or you recommend another V ?
and is there some main change that i should know in this version?

please give me some advice about that 
thank you

**Comment by stof** [comment, marked as answer]
FOSUserBundle works with Symfony 5.4 (if you use the version 3 of the bundle).
However, it is not recommended to use it anymore when starting a new project. See https://github.com/FriendsOfSymfony/FOSUserBundle#maintenance-status

**Reply by mmoqiseh** [reply]
Thank you @stof.
Using FOSUserBundle helped me to control user login session in whole controllers and not making any mistake.
if now symfony 5.4 don't need this bundle and it is possible to handle user by its own. 
can you give me some documents to read about that or tell what to do. cause i'm a little confuse about that.
